src/Adapters/Storage/Postgres/PostgresStorageAdapter.js

```diff
@@ -1397,6 +1451,30 @@ export class PostgresStorageAdapter {
         console.error(error);
       });
   }
+
+  createIndexes(className, indexes, conn) {
+    conn = conn || this._client;
```

Comment:
Better implementation:
```js
    return (conn || this._client).tx(t => t.batch(indexes.map(i => {
        return t.none(`CREATE INDEX $1:name ON $2:name ($3:name)`, [index.name, className, i.key]);
    })));
```

Reply:
Note that filter `:name` is very smart...

Filter `:name`, for `SQL Names`, has a special logic to it, and it can be:

* a simple text string, which is normal, to represent a single column name
* string `*` (asterisks), which is detected automatically
* array of strings, in which case it is considered a list of column names
* any object, to pull all property names automatically.

Reply:
I didn't know about the property names. Thats super helpful.

Reply:
@dplewis   I have just updated documentation for [SQL Names](https://github.com/vitaly-t/pg-promise#sql-names) :wink: